index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/quiz.css">
+	<title>Ike's Portfolio| Software Projects</title>
+</head>
+<body>
+
+	<nav id="nav_bar">
+		<ul>
+			<li><a href="#">Home</a></li>
+			<li><a href="../projects.html">Projects</a></li>
+			<li><a href="../blog.html">Blog</a></li>
+		</ul>
+	</nav>
+<main>
+	<h1>Ike Nwaelele's Portfolio of Software Projects</h1>
+	<img  id="profile_pic" height="400" width="400" alt="Picture of Ike Nwaelele" src="images/Ike_profile_pic.jpg">
+	<article id="bio">
+		<h3>Short Bio</h3>
+		<p>Ike Nwaelele is a Project manager with elite negotiation, communication, and progressive planning skills. Track record of success adapting to IT projects in traditional and agile environments.  Experience includes: building project teams, communicating with senior management, managing business processes, eliciting requirements, and making data-driven decisions, now he's pursuing his dream of becoming more technical so he can be a better leader and contributor on increasingly cooler projects.
```

Comment:
Unrelated to code, but I was a writer/editor in a past life: This uses a mix of full sentences and phrases, which makes it confusing to read. Either write it out in full 'prose' mode or use bullet points or something similar to indicate that you're summarizing your bio info in short chunks. You also have a bit of a run-on where it says "...making data-driven decisions, now he's..." I'd put a period after "decisions" and start a new sentence with "Now".